build/lib/pyshopee/item.py

```diff
@@ -343,6 +343,8 @@ def set_item_installment_tenures(self, **kwargs):
         @@Significant OpenAPI Updates (2019-06-03)
         """
         return self.client.execute("item/installment/set", "POST", kwargs)
+     def get_comment(self, **kwargs):
```

Comment:
Can we add an extra space row above line 346?

Reply:
Yes sure